nautobot/dcim/tests/test_filters.py

```diff
@@ -185,69 +185,49 @@ def common_test_data(cls):
-        site=cls.site,
-    )
-    loc1 = Location.objects.create(
-        name="RTP",
-        location_type=lt1,
-        status=status_active,
-        site=cls.site,
-        description="Research Triangle Park",
-        asn=65003,
-        facility="Facility 3",
-        contact_name="Contact 3",
-        contact_phone="123-555-0003",
-        contact_email="contact3@example.com",
-        time_zone="America/Los_Angeles",
-        physical_address="1 road st, hoboken, nj",
-        shipping_address="PO Box 1, hoboken, nj",
-    )
-    loc2 = Location.objects.create(
-        name="RTP4E",
-        location_type=lt2,
-        status=status_active,
-        parent=loc1,
-        asn=65004,
-        facility="Facility 4",
-        contact_name="Contact 4",
-        contact_phone="123-555-0004",
-        contact_email="contact4@example.com",
-        time_zone="America/Chicago",
-    )
-    loc3 = Location.objects.create(
-        name="RTP4E-3",
-        location_type=lt3,
-        status=status_active,
-        parent=loc2,
-        tenant=tenants[0],
-        asn=65005,
-        latitude=10,
-        longitude=10,
-    )
-    loc4 = Location.objects.create(
-        name="RTP4E-3-0101",
-        location_type=lt4,
-        status=status_active,
-        parent=loc3,
-        tenant=tenants[1],
-        description="Cube",
-        asn=65006,
-        latitude=20,
-        longitude=20,
-        comments="comment2",
-    )
-    nested_loc = Location.objects.create(
-        name="RTP South", location_type=lt1, status=status_active, parent=loc1, asn=65007
-    )
+    loc0 = Location.objects.filter(location_type=lt1).first()
+    loc1 = Location.objects.filter(location_type=lt1).first()
+    loc1.description = "Research Triangle Park"
+    loc1.asn = 65003
+    loc1.facility = "Facility 3"
+    loc1.contact_name = "Contact 3"
+    loc1.contact_phone = "123-555-0003"
+    loc1.contact_email = "contact3@example.com"
+    loc1.time_zone = "America/Los_Angeles"
+    loc1.physical_address = "1 road st, hoboken, nj"
+    loc1.shipping_address = "PO Box 1, hoboken, nj"
+    loc2 = Location.objects.filter(location_type=lt2).first()
+    loc2.parent = loc1
+    loc2.asn = 65004
+    loc2.facility = "Facility 4"
+    loc2.contact_name = "Contact 4"
+    loc2.contact_phone = "123-555-0004"
+    loc2.contact_email = "contact4@example.com"
+    loc2.time_zone = "America/Chicago"
+    loc3 = Location.objects.filter(location_type=lt3).first()
+    loc3.parent = loc2
+    loc3.tenant = tenants[0]
+    loc3.asn = 65005
+    loc3.latitude = 10
+    loc3.longitude = 10
+    loc4 = Location.objects.filter(location_type=lt4).first()
+    loc4.parent = loc3
+    loc4.tenant = tenants[1]
+    loc4.description = "Cube"
+    loc4.asn = 65006
+    loc4.latitude = 20
+    loc4.longitude = 20
+    loc4.comments = "comment2"
+    nested_loc = Location.objects.filter(location_type__nestable=True, parent=loc1).first()
+    nested_loc.asn = 65007
```

Comment:
Can we instead update `LocationFactory` so that it automatically populates these new fields appropriately in all cases? I'm okay if you want to do that as a separate PR, but it's definitely where we want to end up -- should be able to copy a lot of it from `SiteFactory`...